test/MessageTraitTest.php

```diff
@@ -212,8 +216,9 @@ public function invalidHeaderValues()
      */
     public function testWithHeaderRaisesExceptionForInvalidValueType($value)
     {
-        $this->setExpectedException('InvalidArgumentException', 'Invalid header value');
-        $message = $this->message->withHeader('X-Foo', $value);
+        $this->expectException(InvalidArgumentException::class, 'Invalid header value');
```

Comment:
This isn't doing what you think.

`expectException()` only accepts one argument, the exception type to test for. If you also want to test the message, you need to add an additional call to `expectExceptionMessage()`.